apps/frontend/src/components/global/sidebaritems/SidebarFileList.vue

```diff
@@ -91,15 +91,16 @@ export default class FileItem extends mixins(ServerMixin) {
   save_evaluation(file: EvaluationFile) {
     this.saving = true;
 
-    let evaluationDTO: ICreateEvaluation = {
-      data: file.evaluation.data,
-      filename: file.filename,
-      userId: ServerModule.userInfo.id,
-      evaluationTags: []
-    };
+    const formData = new FormData();
+    formData.append("filename", file.filename);
+    var blob = new Blob([JSON.stringify(file.evaluation.data)], {type: 'text/plain'});
+    console.log(blob);
+    formData.append("data", blob);
+    // Seems that publicty hasn't been setup yet
```

Comment:
Is this relevant to include here? This is being handled as part of the parent PR this is referencing.

Reply:
This is just here to make it work, I can remove the comment though